civis_jupyter_notebooks/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.2.2'
+__version__ = '0.2.3'
```

Comment:
Do we want to make this a minor change? 
@beckermr we aren't ready to version these changes yet. A lot more work needs to be done for git integration with notebooks and we don't want to make a new version until all those changes are in place. Should we remove this change and merge into master? Then when we are ready for a new tag we can create a tag and version?

Reply:
Yep. Don't bump the version until we are ready to push to pypi. Then we can just bump the version in a single PR, tag and then push to pypi.

Reply:
Sorry, I had made that version increment in error.  Matt had told me to increment from 0.2.2 to 0.2.3, but the implication I had missed was that I should be merging in master, which contains his recent change to add the terminal.

Though when it does come time to merge this in, we should ask the question of what the versioning strategy here is.  Matt's change went in as an increment to the patch version.

Reply:
Got it. This is not really an issue per say, just a convention. 

You should add an Unreleased entry at the top of the changelog. 

I would say that a new feature that is backwards compatible should result in a minor version change, not a mico. The next bump should be to 2.3